build/dockerfiles/Dockerfile

```diff
@@ -19,6 +19,13 @@ RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
 RUN chmod -R g+rwX /build/devfiles
 
+FROM docker.io/node:12.20.1-alpine3.12 as dwtamplates
```

Comment:
if you add something there, is needs to be added into rhel.Dockerfile as well

Reply:
done